config/wdio/dev/wdio.conf.js

```diff
@@ -0,0 +1,123 @@
+  // logLevel: 'silent',
+  //
+  // Enables colors for log output.
+  coloredLogs: true,
+  //
+  // Saves a screenshot to a given path if a command fails.
+  screenshotPath: './errorShots/',
+  //
+  // Set a base URL in order to shorten url command calls. If your url parameter starts
+  // with "/", the base url gets prepended.
+  // baseUrl: 'http://localhost:5555',
+  //
+  // Default timeout for all waitForXXX commands.
+  waitforTimeout: 10000,
+  //
+  // Initialize the browser instance with a WebdriverIO plugin. The object should have the
+  // plugin name as key and the desired plugin options as property. Make sure you have
+  // the plugin installed before running any tests. The following plugins are currently
+  // available:
+  // WebdriverCSS: https://github.com/webdriverio/webdrivercss
+  // WebdriverRTC: https://github.com/webdriverio/webdriverrtc
+  // Browserevent: https://github.com/webdriverio/browserevent
+  // plugins: {
+  //     webdrivercss: {
+  //         screenshotRoot: 'my-shots',
+  //         failedComparisonsRoot: 'diffs',
+  //         misMatchTolerance: 0.05,
+  //         screenWidth: [320,480,640,1024]
+  //     },
+  //     webdriverrtc: {},
+  //     browserevent: {}
+  // },
+  //
+  // Framework you want to run your specs with.
+  // The following are supported: mocha, jasmine and cucumber
+  // see also: http://webdriver.io/guide/testrunner/frameworks.html
+  //
+  // Make sure you have the node package for the specific framework installed before running
+  // any tests. If not please install the following package:
+  // Mocha: `$ npm install mocha`
+  // Jasmine: `$ npm install jasmine`
+  // Cucumber: `$ npm install cucumber`
+  framework: 'mocha',
+  //
+  // Test reporter for stdout.
+  // The following are supported: dot (default), spec and xunit
+  // see also: http://webdriver.io/guide/testrunner/reporters.html
+  reporter: 'spec',
+
+  //
+  // =====
+  // Hooks
+  // =====
+  // Run functions before or after the test. If one of them returns with a promise, WebdriverIO
+  // will wait until that promise got resolved to continue.
+  // see also: http://webdriver.io/guide/testrunner/hooks.html
+  //
+  // Gets executed before all workers get launched.
+  // onPrepare: function() {
+  //     // do something
+  // },
+  //
+  // Gets executed before test execution begins. At this point you will have access to all global
+  // variables like `browser`. It is the perfect place to define custom commands.
+  // before: function() {
+  //     // do something
+  // },
+  //
+  // Gets executed after all tests are done. You still have access to all global variables from
+  // the test.
+  // after: function() {
+  //     // do something
+  // },
+  //
+  // Gets executed after all workers got shut down and the process is about to exit. It is not
+  // possible to defer the end of the process using a promise.
+  // onComplete: function() {
+  //     // do something
+  // }
+};

```

Comment:
As it is a config file, this is interesting to keep those unused blocks in case we would like to use them afterwards :)